src/server/cmd/mount-server/cmd/cmd.go

```diff
@@ -34,6 +36,6 @@ func MountServerCmd() *cobra.Command {
 	}
 	rootCmd.Flags().StringVar(&mountDir, "mount-dir", "/pfs", "Target directory for mounts e.g /pfs")
 	rootCmd.Flags().BoolVar(&allowOther, "allow-other", true, "Mount is created with allow-other option")
-
+	rootCmd.Flags().StringVar(&sockPath, "sock-path", "", "Socket path to be used, default set to empty")
```

Comment:
Default is to listen on http port 9002

Reply:
The default is being set in the env.py, currently it is set to use unix domain socket. It can be adjusted using the environment when the container is launched.

Reply:
| The default is being set in the env.py, currently it is set to use unix domain socket. It can be adjusted using the environment when the container is launched.

No, that tells the mount client to add --sock-path.   This is the CLI of the mount-server -- by default it still listens no port 9002.

Reply:
Understood, so if I run the command `mount-server` with no options it will use http port 9002, so do we want to run it using unix domain socket in such a case? I could set the default value to "/tmp" and that will change the default behavior to use uds

Reply:
| Understood, so if I run the command mount-server with no options it will use http port 9002, so do we want to run it using unix domain socket in such a case? 

No.  That would not be an upward compatible change in the mount-server interface, that is why it is an option.
On the mount_server_client.py side, if it is working robustly, it would not necessarily need to be configurable there with an env var as you have.   That does make testing easier though, so not a bad thing for now.